cluster-provision/manifests/cni/calico/script.d/003-change-cidr.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/env bash
+
+set -e
+
+cni_name=$1
+default_cidr="192.168.0.0/16"
```

Comment:
I think $pod_cidr is missing

Reply:
all the vars tat config-cni.sh are usable here too, since this scripts is called from there.